news/84.new.rst

```diff
@@ -0,0 +1 @@
+Added a hook for the 'pygraphviz' library
```

Comment:
This is PR #86 - not 84. The file should be named as such.

Reply:
Ah, I thought this is just a counter. :-) Fixed now.